src/Equations/Fluid/HottailRateTermLowZ.cpp

```diff
@@ -0,0 +1,423 @@
+/**
+ * Implementation of equation term representing the runaway generation rate
+ * due to hottail when using an analytic distribution function
```

Comment:
Could you add mention in this comment that this module implements the generation rate from section 4.1 of Ida Svenningsson's MSc thesis (and maybe provide the link https://hdl.handle.net/20.500.12380/300899 ?)